deploy/runtime/k3d/helm/templates/feedercan.yaml

```diff
@@ -46,7 +46,7 @@ spec:
             - name: VEHICLEDATABROKER_DAPR_APP_ID
               value: {{.Values.imageVehicleDataBroker.daprAppid}}
             - name: LOG_LEVEL
-              value: "info,databroker=info,dbcfeeder.broker_client=debug,dbcfeeder=debug"
+              value: "info,databroker=info,dbcfeeder.broker_client=info,dbcfeeder=info"
```

Comment:
Looks good to me, however did you try that? 

When you run feedercan, there should no longer be repeating logs. Please verify.

Reply:
Yes, it is tested and there were no repeating logs